README.md

```diff
@@ -197,6 +197,16 @@ this API too. In addition, the parameter `iceTransportPolicy=relay` will force c
 generating host and server-reflexive ICE candidates and
 use TURN for connecting unconditionally. This will make client connections much faster.
 
+### Ensuring valid Gateway public IP addresses
+
+In certain scenarios, STUNner is unable to determine the public IP for a Gateway
+(e.g. private Kubernetes clusters without LoadBalancer and node ExternalIPs). In
+these cases the public IP can be manually set using these methods, in order of precedence:
+
+- Setting the [`public-ip` URL parameter](#request) when requesting configurations.
+- Setting the `STUNNER_PUBLIC_IP` environment variable (see the
+    "stunner-auth-server" container in the [Kubernetes manifest](deploy/kubernetes-stunner-auth-service.yaml)).
```

Comment:
Can we please switch the order here? I mean, so that the environment is less preferred than whatever is explicitly requested by the user.

Reply:
I have changed the wording. I had it right in my mind (it's logical that the URL parameter takes precedence over the environment), but failed to express it correctly. Now I have:

```
...can be manually set using the following methods, listed in
order of priority (from highest to lowest):

- URL param
- env

If multiple methods are used, the one with the highest priority will override the rest
(e.g., setting the `public-ip` parameter always takes precedence).
```
How does this look?